client/me/profile/index.jsx

```diff
@@ -113,7 +113,7 @@ const Profile = createReactClass( {
 						</FormFieldset>
 
 						<FormFieldset>
-							<FormLabel htmlFor="description">{ this.props.translate( 'About Me' ) }</FormLabel>
+							<FormLabel htmlFor="description">{ this.props.translate( 'About me' ) }</FormLabel>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.